layout-server/src/test/java/org/finos/vuu/layoutserver/controller/ApplicationLayoutControllerTest.java

```diff
@@ -27,17 +27,16 @@ public void setup() {
     }
 
     @Test
-    public void getApplicationLayout_anyUsername_returnsLayoutFromService() throws JsonProcessingException {
+    public void getApplicationLayout_anyUsername_returnsLayoutDefinitionFromService() throws JsonProcessingException {
```

Comment:
My IDE is pointing out that the throws list is no longer required as `JsonProcessingException` isn't thrown.

Reply:
done :)